### ISSUE TITLE
AVT 3 - React Pagination Component, iOS/VoiceOver navigation issues in dropdown and on images

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3451]
## Environment
iOS 12.3.1
Safari
VoiceOver

## Description
1. Swipe R to the items per page dropdown, VO announces the number of items per page as expected.
2. Double-tap to activate the picker, VO doesn't say anything.
3. Swipe Right and nothing happens
4. Swipe Down and VO announces, "Form control not found"
5. Swipe Left and VO announces the last select in the drop-down.
6. Make a select other than 10, Focus moves out of pagination. 
7. Tab back to the first pagination dropdown and then swipe right, VO announces "open list of options image".
Note: Same issue exists when changing the number per page.
**Expected Result:** Swipe R should navigation down the list and when a select is made user should remain where they left off. When reading across the pagination the images should be ignored by the screen reader.
